apps/site/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import Image from "next/image";
+
+import styles from "./Footer.module.scss";
+
+import Mail from "@/assets/icons/mail.svg";
+import Facebook from "@/assets/icons/facebook.svg";
+import Instagram from "@/assets/icons/insta.svg";
+import Twitter from "@/assets/icons/twitter.svg";
+
+const SOCIALS = [
+	{
+		icon: Mail,
+		link: "mailto:hack@uci.edu",
+		alt: "Mail",
+		width: 78,
+		height: 78,
+		marginRight: "25px",
+	},
+	{
+		icon: Facebook,
+		link: "https://www.facebook.com/UCI.Hack/",
+		alt: "Facebook",
+		width: 59,
+		height: 59,
+		marginRight: "16px",
+	},
+	{
+		icon: Instagram,
+		link: "https://www.instagram.com/hackatuci/",
+		alt: "Instagram",
+		width: 91,
+		height: 91,
+		marginRight: "16px",
+	},
+	{
+		icon: Twitter,
+		link: "https://www.twitter.com/HackUCI",
+		alt: "Twitter",
+		width: 64,
+		height: 53,
+		marginRight: "0px",
+	},
+];
```

Comment:
I suppose, for conciseness, you could create your own `Social` type and then declare `SOCIALS` as type `Social[]`. Example: https://www.digitalocean.com/community/tutorials/how-to-create-custom-types-in-typescript